smodels/tools/simplifiedLikelihoods.py

```diff
@@ -891,6 +894,22 @@ def likelihood(self, nsig, marginalize=False, nll=False, mu=None):
         else:
             return self.profileLikelihood(nsig, nll)
 
+    def likelihood_beta(self, mu: float, marginalize: bool = False, nll: bool = False) -> float:
```

Comment:
@WolfgangWaltenberger once you are happy we can move beta to likelihood.

Reply:
@jackaraz unittests run through for me, am happy with the commit, only comment i have is about the comment:
simplifiedLikelihoods.py:898
compute likelihood for nsig, profiling the nuisances    -> compute likelihood for mu, profiling or marginalizing the nuisances